P/PROJ/build_tarballs.jl

```diff
@@ -0,0 +1,85 @@
+using BinaryBuilder
+
+name = "PROJ"
+version = v"6.2.1"
+
+# Collection of sources required to build PROJ
+sources = [
+    "https://download.osgeo.org/proj/proj-$version.tar.gz" =>
+    "7f2e0fe63312f1e766057cceb53dc9585c4a335ff6641de45696dbd40d17c340",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/proj-*/
+
+# sqlite needed to build proj.db, so this should not be the
+# cross-compiled one since it needs to be executed on the host
+apk add sqlite
+
+if [[ ${target} == *mingw* ]]; then
+    SQLITE3_LIBRARY=${libdir}/libsqlite3-0.dll
+else
+    SQLITE3_LIBRARY=${libdir}/libsqlite3.${dlext}
+fi
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DSQLITE3_INCLUDE_DIR=$prefix/include \
+      -DSQLITE3_LIBRARY=$SQLITE3_LIBRARY \
+      -DHAVE_PTHREAD_MUTEX_RECURSIVE_DEFN=1 \
+      -DBUILD_LIBPROJ_SHARED=ON \
+      ..
+cmake --build .
+make install
+
+# add proj-datumgrid files directly to the result
+wget https://download.osgeo.org/proj/proj-datumgrid-1.8.tar.gz
+tar xzf proj-datumgrid-1.8.tar.gz -C $prefix/share/proj/
+"""
+
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libproj", :libproj),
```

Comment:
It looks like you need to add `libproj_6_2` as additional searched name, for Windows